server/src/main/java/org/opensearch/action/search/DeletePitAction.java

```diff
@@ -17,7 +17,7 @@
 public class DeletePitAction extends ActionType<DeletePitResponse> {
 
     public static final DeletePitAction INSTANCE = new DeletePitAction();
-    public static final String NAME = "indices:admin/read/pit/delete";
+    public static final String NAME = "indices:data/read/point_in_time/delete";
```

Comment:
Please create a separate PR with create PIT and delete PIT changes. It would be smaller PR to review. This will help reviewers to not change the context. (I haven't started the review yet. I will wait for you to make this change)